README.md

```diff
@@ -22,10 +22,17 @@ This GitHub action checks all Markdown files in your repository for broken links
 
 ## Configuration
 
-- [Custom variables](#custom-variables)
-- [Scheduled runs](#scheduled-runs)
-- [Disable check for some links](#disable-check-for-some-links)
-- [Check only modified files in a pull request](#check-only-modified-files-in-a-pull-request)
+- [GitHub Action - Markdown link check 🔗✔️](#github-action---markdown-link-check-%f0%9f%94%97%e2%9c%94%ef%b8%8f)
```

Comment:
I would assume that the TOC was intended to list only the configurable items - and not a complete TOC. Suggestion: remove the change.